crates/tabby/src/serve.rs

```diff
@@ -189,6 +194,10 @@ async fn api_router(
                 "/v1/health",
                 routing::get(routes::health).with_state(health_state),
             )
+            .route(
+            "/v1/metrics",
```

Comment:
1. Make the route listen to `/metrics`
2. Add the route to `worker.rs` as well

Reply:
Thanks for the review!

There were some changes merged in the meantime which introduced some additional work, let me know if that's what you had in mind